sale_loyalty_discount_attribute/models/sale_order.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _get_reward_values_discount(self, reward, coupon, **kwargs):
```

Comment:
@diogocsc Can you please add a test case for this?

Reply:
Added unit tests in later commit